src/osvr/Server/ConfigFilePaths.h.cmake_in

```diff
@@ -0,0 +1,57 @@
+/** @file
+    @brief Auto-generated configuration header - do not edit!
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_ConfigFilePaths_h_GUID_241E9C9C_0E0E_46B0_9DED_8F8059306192
+#define INCLUDED_ConfigFilePaths_h_GUID_241E9C9C_0E0E_46B0_9DED_8F8059306192
+
+#include <osvr/Util/PlatformConfig.h>
+namespace osvr {
+namespace server {
+
+    std::vector<std::string> getDefaultConfigFilePaths() {
+
+#ifdef OSVR_WINDOWS
+        std::string appdata = std::getenv("APPDATA");
```

Comment:
I wrote a "friendly" wrapper for getenv at least in part to quiet Visual Studio about ownership and unify behavior if an environment variable is not set: https://github.com/OSVR/OSVR-Core/blob/master/inc/osvr/Common/GetEnvironmentVariable.h